front/tests/pages/rolling-stock-selector-page.ts

```diff
@@ -40,6 +40,14 @@ export default class RollingStockSelectorPage extends BasePage {
 
   readonly getNoRollingStockResult: Locator;
 
+  readonly comfortHeatingRadio: Locator;
+
+  readonly comfortACRadio: Locator;
```

Comment:
You make a valid point about consistency in naming conventions. I think adding a suffix improves clarity and helps in quickly identifying the type of element. However, sometimes it might be unnecessary if the name is already clear.

For this test, I got confused because both `radio` and `button` were used in the div.

Therefore, I updated the `Locator` to `Button` .

Maybe I will update all the locators in the tests later to use the same convention